bindata/networking-console-plugin/001-config-map.yaml

```diff
@@ -9,6 +9,7 @@ data:
       keepalive_timeout  65;
       server {
         listen              9443 ssl;
+        listen              [::]:9443 ssl;
```

Comment:
The code below is currently not doing anything, do you want to address that as a part of this PR?
https://github.com/openshift/cluster-network-operator/blob/a8648b35c0e45671be50c1171549e2b28121f35d/bindata/networking-console-plugin/002-deployment.yaml#L64-L70

Reply:
sure but the code I highlighted is not doing anything. In case you don't want to configure the IP family dynamically you should just remove it. It can be done as a followup.